src/components/UserActions/ClickedDocumentList.ts

```diff
@@ -19,7 +8,7 @@ import './Strings';
 /**
  * Initialization options of the **ClickedDocumentList** class.
  */
-export interface IClickedDocumentList {
+export interface IClickedDocumentListOptions {
```

Comment:
Renaming this interface is a breaking change. I understand the new name is better and this library has not yet reached version 1.0.

However, it might still cause issues to developers when upgrading.

I'd suggest to either:

1. revert the change to make it non-breaking
2. or, make sure to document the breaking change in the merge commit description. Hopefully, it would help a developer facing the issue to fix the code.

Reply:
oh yea very true, I'll just revert that change 👍 
I don't put enough value in changing the name I think to deserve creating a breaking change.